README.md

```diff
@@ -4,7 +4,7 @@
 NachoSpigot offers a number of enhancements to performance as well as bug fixes such as a built-in anti-crash system and being able to perform well with a large number of players.
 
 ## Current State
-Java 15 is now natively supported, and [ProtocolLib](https://github.com/dmulloy2/ProtocolLib), [ViaVersion](https://github.com/ViaVersion/ViaVersion), and [Citizens2](https://github.com/CitizensDev/Citizens2) are patched at runtime to work with Nacho's modifications.
+Java 15 is now natively supported, and [ViaVersion](https://github.com/ViaVersion/ViaVersion) is patched at runtime to work with Nacho's modifications.
```

Comment:
ViaVersion isn't patched, we just use some Reflection magic on it.
So that whole sentence can go, you can change it to this though:
```suggestion
Java 15 is natively supported and also the recommended version to use.
```